src/utilities/validateUrl.ts

```diff
@@ -1,5 +1,60 @@
 import { TextFieldSingleValidation } from 'payload'
-import isAbsoluteUrl from './isAbsoluteUrl'
 
-export const validateExternalUrl: TextFieldSingleValidation = (val) =>
-  isAbsoluteUrl(val) || 'URL must be an absolute url with a protocol. I.e. https://www.example.com.'
+/**
+ * Validates whether a string is a valid, full URL with protocol, host, and proper TLD
+ */
+const isValidFullUrl = (url?: string | null): boolean => {
+  if (url == null || typeof url !== 'string' || url.trim() === '') {
+    return false
```

Comment:
I think all of these `false` returns need to be a string. I see the || at the bottom but it might be nice to return some specific errors where necessary.